src/Castle.Core.Tests/Castle.Core.Tests.csproj

```diff
@@ -50,12 +51,14 @@
 
 	<ItemGroup>
 		<ProjectReference Include="..\Castle.Core\Castle.Core.csproj" />
+		<ProjectReference Include="..\Castle.Services.Logging.EventLogIntegration\Castle.Services.Logging.EventLogIntegration.csproj" />
 		<ProjectReference Include="..\Castle.Services.Logging.log4netIntegration\Castle.Services.Logging.log4netIntegration.csproj" />
 		<ProjectReference Include="..\Castle.Services.Logging.NLogIntegration\Castle.Services.Logging.NLogIntegration.csproj" />
 		<ProjectReference Include="..\Castle.Services.Logging.SerilogIntegration\Castle.Services.Logging.SerilogIntegration.csproj" />
 	</ItemGroup>
 	
 	<ItemGroup Condition="'$(TargetFramework)'=='net8.0'">
+    <PackageReference Include="System.Diagnostics.EventLog" Version="8.0.2" />
```

Comment:
In the new `Castle.Services.Logging.EventLogIntegration.csproj` file, you're using a different condition:

```csharp
<ItemGroup Condition=" '$(TargetFrameworkIdentifier)' != '.NETFramework' ">
	<PackageReference Include="System.Diagnostics.EventLog" Version="8.0.2" />
</ItemGroup>
```

Should we perhaps do it the same way here, too?

Reply:
Have changed to `'$(TargetFrameworkIdentifier)' != '.NETFramework'`